library/CMakeLists.txt

```diff
@@ -82,8 +82,52 @@ function( rocblas_library_settings lib_target_ )
     target_link_libraries( ${lib_target_} PRIVATE hip::device )
   else()
     target_link_libraries( ${lib_target_} PRIVATE hip::device -lstdc++fs --rtlib=compiler-rt --unwindlib=libgcc)
-    if (BUILD_SHARED_LIBS)
-      target_link_libraries(${lib_target_} PRIVATE -lroctx64)
+
+    if(BUILD_SHARED_LIBS)
```

Comment:
Looks fine, in future maybe WIN32 and static can just use the DISABLE_ROCTX define.